app/src/main/res/layout/fragment_home_driver.xml

```diff
@@ -11,9 +11,18 @@
         android:layout_height="match_parent"
         tools:context=".activities.DriverActivity" />
     <Button
+        style="@style/Widget.AppCompat.Button"
+        android:id="@+id/findparkingbutton"
         android:text="@string/find_spot_button_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:background="@drawable/rounded_button"
+        android:textColor="@android:color/white"
+        android:textSize="24sp"
+        android:layout_width="match_parent"
+        android:layout_height="64dp"
         android:layout_centerHorizontal="true"
-        android:layout_alignParentBottom="true" />
+        android:layout_alignParentBottom="true"
+        android:textAllCaps="false"
+        android:layout_marginLeft="4dp"
```

Comment:
Haha okay there very well might be but I'm not aware of it / google is not making its existence very apparent